app/views/foreman_tasks/tasks/dashboard/_tasks_status.html.erb

```diff
@@ -0,0 +1,2 @@
+<h4 class="header"><%=_("Task Status")%></h4>
+<%= render :partial => '/foreman_tasks/tasks/dashboard/task_status_table' %>
```

Comment:
Any reason for having this in the separate partial? Not big deal, but having it in one file, unless to many things, is usually a bit more readable